common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -79,7 +79,7 @@ def do_my_expensive_update(course_key):
        do the actual work.
 
     """
-    course_published = django.dispatch.Signal(providing_args=["course_key"])
+    course_published = django.dispatch.Signal(providing_args=["course_key", "item_keys"])
```

Comment:
Yes I can - but I think this is going to change - I am toying with creating a whole separate event. The event as existed prior to this fires any time one updates any item, which is more often then we need.

I'd love any opinions that @ormsbee or @clintonb have on this.